packages/calendar/core/src/constants.ts

```diff
@@ -1,3 +1,17 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+import { Language } from "@prisma/client";
+// @ts-ignore
+import flagDeIcon from "@agreeto/ui/src/assets/flag-de.svg";
+// @ts-ignore
+import flagEsIcon from "@agreeto/ui/src/assets/flag-es.svg";
+// @ts-ignore
+import flagFrIcon from "@agreeto/ui/src/assets/flag-fr.svg";
+// @ts-ignore
+import flagItIcon from "@agreeto/ui/src/assets/flag-it.svg";
+// @ts-ignore
+import flagUSIcon from "@agreeto/ui/src/assets/flag-us.svg";
```

Comment:
These disables/ignores can be avoided when you export a react component from the `@agreeto/ui` package
<img width="416" alt="CleanShot 2022-11-29 at 21 39 46@2x" src="https://user-images.githubusercontent.com/18185649/204642970-bbea5900-b475-459f-92fa-bcd03bb6dc70.png">

Basically like we're [already doing](https://github.com/p6l-richard/create-t3-turbo/blob/19b4aff33e0a58b1dfc6fe9ce80bdc59bd42e3da/packages/ui/src/logo/google.tsx#L3) for the other icons.

Reply:
I went for a lib for these but we could create components like the GoogleIcon for these too

https://github.com/agreeto/agreeto/blob/7ff64b9dfed4fb34dde184b9870ea1311624ddd7/packages/calendar/react/src/components/action-pane/availability.tsx#L66-L69

or 

https://github.com/agreeto/agreeto/blob/7ff64b9dfed4fb34dde184b9870ea1311624ddd7/packages/ui/src/logo/google.tsx#L3